lib/stack_master/cli.rb

```diff
@@ -174,22 +174,21 @@ def load_config(file)
       exit 1
     end
 
-    def execute_stack_command(command, args, options)
-      unless args.size == 2
-        say "Invalid arguments. stack_master #{command.name.split('::').last.downcase} [region] [stack_name]"
-        return
-      end
+    def execute_stacks_command(command, args, options)
       config = load_config(options.config)
-      aliased_region, stack_name = args
-      region = Utils.underscore_to_hyphen(config.unalias_region(aliased_region))
-      stack_name = Utils.underscore_to_hyphen(stack_name)
-      StackMaster.cloud_formation_driver.set_region(region)
-      stack_definition ||= config.find_stack(region, stack_name)
-      if stack_definition.nil?
-        say "Could not find stack definition #{stack_name} in region #{region}"
-        return
+      args = [nil, nil] if args.size == 0
+      args.each_slice(2) do |aliased_region, stack_name|
```

Comment:
Could be a bit ambiguous with the the region names vs stack names.

Maybe something like

`stack_master apply us-east-1 stack-1,stack-2,stack-3`